docs/getting-started.md

```diff
@@ -15,6 +19,21 @@ You may need to download the [CoreOS Application Signing Key](http://coreos.com/
 
 New releases of `fcct` are backwards compatible with old releases unless otherwise noted.
 
+#### Container
+
+This example uses podman, but docker can also be used. Substitute v0.2.0 with the desired version. Note that the `latest` tag corresponds with `master` and not with the latest release.
+
+```bash
+# Pull the desired version
+podman pull quay.io/coreos/fcct:v0.2.0
+
+# Run fcct using standard in and standard out
+podman run -it --rm fcct -pretty -strict < your_config.fcc > transpiled_config.ign
```

Comment:
Hmm, doesn't this need to be the full pullspec again? So e.g. `podman run -it --rm quay.io/coreos/fcct:v0.2.0 ...`?

Reply:
Not sure, but having it doesn't hurt and is definitely more clear. Fixed